activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-core/src/main/java/org/activiti/cloud/services/core/VariablesPayloadConverter.java

```diff
@@ -74,19 +109,21 @@ private Map.Entry<String, Object> parseValue(Map.Entry<String, Object> entry) {
         Object entryValue = entry.getValue();
 
         try {
-            if (Map.class.isInstance(entryValue)) {
-                Map<String, String> valuesMap = Map.class.cast(entryValue);
+            if (entryValue instanceof Map) {
+                Map<String, String> valuesMap = (Map) entryValue;
 
                 if (valuesMap.containsKey("type") && valuesMap.containsKey("value")) {
                     String type = valuesMap.get("type");
                     String value = valuesMap.get("value");
 
                     entryValue = variableValueConverter.convert(new ProcessVariableValue(type, value));
                 }
-            } else if (ProcessVariableValue.class.isInstance(entryValue)) {
-                entryValue = variableValueConverter.convert(ProcessVariableValue.class.cast(entryValue));
+            } else if (entryValue instanceof ProcessVariableValue) {
```

Comment:
You can use Java inline type cast here `if (entryValue instanceof ProcessVariableValue processVariableValue)`

Reply:
Nice! Thanks!